test/recipes/70-test_sslmessages.t

```diff
@@ -169,12 +169,13 @@ my $proxy = TLSProxy::Proxy->new(
     [0,0,0,0]
 );
 
+plan tests => 21;
+
 #Test 1: Check we get all the right messages for a default handshake
 (undef, my $session) = tempfile();
 $proxy->serverconnects(2);
 $proxy->clientflags("-no_tls1_3 -sess_out ".$session);
 $proxy->start() or plan skip_all => "Unable to start up Proxy for tests";
-plan tests => 21;
```

Comment:
Please don't make this change. The plan is deliberately done late so that it appears after `$proxy->start()` (we had some platforms where the proxy wouldn't start due to environmental issues and we didn't want the test to fail)